fearless/Common/Helpers/FWKeychainManager.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+import SoraKeystore
+
+final class FWKeychainManager {
```

Comment:
please modify this class in common repository. just pass queue to constructor e.g.